### PR TITLE
Add ListCommand closes #7

### LINKED ISSUE #7: List
As the logistics in-charge of a pharmacy, I can list out all the medicine currently stocked, so that I can get an overview of what the pharmacy has at the moment.

### PR DESCRIPTION
**Pull request by cnivedit**
Create ListCommand class with empty constructor, execute method overrides baseclass
Add JUnit tests for empty list and sample test cases

### CODE REVIEW

File: src/main/java/seedu/pill/command/ListCommand.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package seedu.pill.command;
+
+import seedu.pill.util.ItemList;
+
+public class ListCommand extends Command{
+    public ListCommand() {}
+
+    @Override
```
Comment (yijiano) [inline src/main/java/seedu/pill/command/ListCommand.java:8]: Good work here, clean and concise code. Maybe you can add JavaDocs next time.

File: src/test/java/seedu/pill/command/ListCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        PrintStream printStream = new PrintStream(outputStream);
+        System.setOut(printStream);
+
+        // Test command
+        listCommand.execute(itemList);
+
+        //Compare output
+        String output = outputStream.toString();
+        assertEquals(output, expectedOutput);
```
Comment (yijiano) [inline src/test/java/seedu/pill/command/ListCommandTest.java:58]: Good work on the JUnit test implementation for ListItems!

**Review by yijiano — APPROVED** [review]
LGTM
